### PR TITLE
`Write_eml()`

### LINKED ISSUE #61: Add `write_eml()`
Hi! While working on camtraptor version 1.0, I am going to remove `write_dwc()` (and associated file `write_dwc.R`) in camtraptor as this functionality has been transferred to camtrapdp, see #55. 

Questions:
1. Can I remove function from camtraptor as well?
2. Will `write_eml()` be implemented in camtrapdp as well?

### PR DESCRIPTION
**Pull request by sannegovaert**
Fix #61 

Question: where or how can users provide or change metadata, e.g. keywords or the (order of) authors?

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -27,10 +27,14 @@ BugReports: https://github.com/inbo/camtrapdp/issues
...
     cli,
     dplyr,
+    EML,
     frictionless (>= 1.1.0),
     memoise,
     purrr,
-    readr
+    readr,
+    rlang,
+    tidyr,
```
Comment (peterdesmet) [inline DESCRIPTION:36]: @sannegovaert This is not a light dependency and I notice we only use the `separate()` function. Is it possible to refactor this, so we don't need tidyr?

Comment (sannegovaert) [reply DESCRIPTION]: Done

**Review by peterdesmet — APPROVED** [review]
Nice! 🎉

- The updated the tests, so they are similar to movepub. It now compares with the actual written `eml.xml` file using `expect_snapshot_file()` (this requires the helper function `remove_uuid()`) rather than the `eml.md` output generated by by `expect_snapshot()`
- I added the same `derived_paragraph` parameter + added tests
- I replaced the `split_first_space()` helper function with `sub()` and `strsplit()`

**Comment by peterdesmet** [comment]
> where or how can users provide or change metadata, e.g. keywords or the (order of) authors?

By updating the camtrapdp object. Not super straightforward atm, but not impossible. Could be facilitated in frictionless

**Pull request merged**